week-1-ts-ecosystem/w01e03-object-oriented-structure/src/shop/product.interface.ts

```diff
@@ -0,0 +1,14 @@
+interface CommonInterface {
```

Comment:
Do nazw modeli danych nie musisz dodawać `*Inteface`  jako słowo.

Wydaje mi sie że ten `common` to zbyt ogólna nazwa.
Może coś jako `BaseProduct` - w ten sposób ?